src/meta/processors/job/BalanceJobExecutor.h

```diff
@@ -8,20 +8,44 @@
 
 #include "meta/processors/job/BalancePlan.h"
 #include "meta/processors/job/BalanceTask.h"
+#include "meta/processors/job/MetaJobExecutor.h"
 #include "meta/processors/job/SimpleConcurrentJobExecutor.h"
 
 namespace nebula {
 namespace meta {
-
-using ZoneParts = std::pair<std::string, std::vector<PartitionID>>;
 using HostParts = std::unordered_map<HostAddr, std::vector<PartitionID>>;
-using PartAllocation = std::unordered_map<PartitionID, std::vector<HostAddr>>;
 using LeaderBalancePlan = std::vector<std::tuple<GraphSpaceID, PartitionID, HostAddr, HostAddr>>;
-using ZoneNameAndParts = std::pair<std::string, std::vector<PartitionID>>;
 
-class BalanceJobExecutor : public MetaJobExecutor {
-  friend void testRestBlancer();
+struct Host {
+  explicit Host(const HostAddr& ha) : ha_(ha) {}
+  Host() = default;
+
+  HostAddr ha_;
```

Comment:
ha is a confused abbreviation, it usually means High Available. host_ or addr_ may be better.

Reply:
> ha is a confused abbreviation, it usually means High Available. host_ or addr_ may be better. Please check all the ha when modify it.

ok done